src/modules/site-v2/base/static/js/plot.js

```diff
@@ -79,7 +80,7 @@ function add_histogram_along_axis(d, axis, data, target, config={}) {
   const bins = d3.histogram()
     .domain(axis.domain())
     .thresholds(axis.ticks(num_bins))
-    .value(n => n[d])
+    .value(n => n[d + 1])
```

Comment:
consider adding a comment about the need for `+1`, or create a named variable that hints at why we need to offset the index.